src/pages/create-or-edit-lesson/CreateOrEditLesson.tsx

```diff
@@ -125,32 +129,18 @@ const CreateOrEditLesson = () => {
     handleNonInputValueChange('content', content)
   }
 
-  const addLesson = (): Promise<AxiosResponse> => {
+  const addLesson = (): Promise<Lesson> => {
```

Comment:
It's redundant here, this function is typed automatically
```suggestion
  const addLesson = () => {
```